lib/matplotlib/axes/_axes.py

```diff
@@ -2350,17 +2350,19 @@ def stem(self, *args, linefmt=None, markerfmt=None, basefmt=None,
             which inspired this method.
 
         """
+        if len(args) > 5:
```

Comment:
You may as well do
```
if not 1 <= len(args) <= 5:
    raise TypeError("stem expected between 1 and 5 positional arguments, got {}".format(len(args))
```
which lets you get rid of the comment immediately below.

Reply:
Done.